### ISSUE TITLE
some questions about the result

### ISSUE THREAD

**Issue by challow0** [DocF/multispectral-object-detection#24]
Thanks for your great work~ I find the result in LLVIP origin paper is higher than the result in your paper. I want to know why this difference happened. I did the experiment using the `yolov5 v6.0 version` and found it's the result in LLVIP paper. (**AP 0.527 and 0.67**). Can you help me to explain this question? ^_^ 
- The result in LLVIP paper (using yolov5)

![image](https://user-images.githubusercontent.com/46772513/161688349-cbf33792-cddf-4052-a624-2544af1fe96b.png)

- The result in your paper 

![image](https://user-images.githubusercontent.com/46772513/161688648-2b42cf7b-8de0-4214-97ad-519ce0d653a9.png)

**Comment by DocF** [comment]
Yes, the detection results on the single modality of the LLVIP dataset come from the LLVIP paper.
By the way, the baseline of this repository is `yolov5 v5.0 version`.

**Comment by yeahyeahye** [comment]
您好，我也想问一下就是为什么您论文中关于RGB和IR的结果跟LLVIP论文中的结果不一致呀，如果IR的结果都优于多光谱融合后的检测结果了，那做多光谱的意义是什么

**Comment by Hiram1026** [comment]
会不会是因为作者用的1024img_size丢了信息，LLVIP paper用的1280

**Comment by yeahyeahye** [comment]
有道理唉，我之前都没注意到这个



---Original---
From: ***@***.***&gt;
Date: Fri, Apr 22, 2022 20:48 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [DocF/multispectral-object-detection] some questions about the result (Issue #24)




 
会不会是因为作者用的1024img_size丢了信息，LLVIP paper用的1280

**Comment by Hiram1026** [comment]
![image](https://user-images.githubusercontent.com/86820620/165224557-7321b169-6829-4aa5-b87d-9b8016633c9a.png)
![image](https://user-images.githubusercontent.com/86820620/165224641-386f5603-5240-4b2d-bc62-e245ebf4f5a6.png)
@DocF 这是我在yolov5上跑的红外单模态，为什么IR的指标都优于了融合后
